src/caisy/entities.ts

```diff
@@ -0,0 +1,44 @@
+import { normalizeList } from "./utils";
+
+export const getEntitiesData = async (params: {
+  projectId: string,
+  query: string,
+}
+) => {
+  const { projectId, query } = params
+  const url = `https://cloud.caisy.io/api/v3/e/${projectId}/graphql`;
```

Comment:
This can be extracted out since the same `https` url is re-repeating. Some func like `getURlWithProject` something like that